source/deployment/auth.rst

```diff
@@ -1,28 +1,34 @@
 ..  _auth_mfa:
+
 Multi-factor Authentication
 ===========================
 
-For organizations that require multi-factor authentication (MFA) as part of their IT security policy, and choose not to secure Mattermost behind a firewall with their existing MFA infrastructure, Mattermost offers a smartphone-based MFA check, in addition to email-password or Active Directory/LDAP authentication, to sign in to the Mattermost server.
+Multi-factor authentication (MFA) is a secondary layer of user authentication applied to your Mattermost server.
+
+With MFA-enabled, users need to provide a secure one-time code generated by an app or smartphone in addition to their username and password in order to log in to Mattermost. MFA is useful in organizations that have specific security and compliance policies. It can also be used in organizations where Mattermost is not behind a firewall and doesn't have access to existing MFA infrastructure.
```

Comment:
```suggestion
With MFA enabled, users need to provide a secure one-time code in addition to their username and password in order to log in to Mattermost. MFA is useful in organizations that have specific security and compliance policies. It can also be used in organizations where Mattermost is not behind a firewall and doesn't have access to existing MFA infrastructure.
```

```suggestion
With MFA enabled, users provide a secure one-time code in addition to their username and password in order to log in to Mattermost. Some IT security policies require MFA, and it can improve security for organizations that choose not to secure Mattermost behind a firewall.
```